docs/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.md

```diff
@@ -6,9 +6,178 @@ jupyter:
+
+#### `ggplot` API Boxplot
+
+To keep a user engaged and interested in your dashboard, it is important to use different libraries too along with visualizations and widgets. This dashboard also makes use of `JupySQL`s `ggplot` API, and the second visualization is a boxplot of fuel consumption and $CO_2$ emissions of all types of vehicles created from it. Recall this [example](https://ploomber-sql.readthedocs.io/en/latest/visualizing-your-sql-queries/plotting-with-ggplot.html#question-2-easy) from the Visualizing Your SQL Queries module.
+
+The boxplots can be visualized either individually or together with the use of the `SelectMultiple` widget. There are three fuel consumption variables in the dataset, one for city driving, one for highway driving, and one for combined driving. The `SelectMultiple` widget allows the user to select one or more of these variables to visualize in the boxplot. However, the $CO_2$ emissions variable, `co2emissions_g_km`, is <b>measured in a different scale</b>, grams per kilometer, than the fuel consumption variables scale, liters per 100 kilometers. Therefore, <b>it is recommended to visualize `co2emissions_g_km` individually</b> to avoid confusion.
+
+The process of building the boxplot is slightly different to the bar plot. A CTE is created from the `all_vehicles` table, but <b>not</b> converted into a `pandas` DataFrame because  the `ggplot` function takes in only a `SQL` table name, instead of a dataset. Then, the `SelectMultiple` widget is stored in a variable `columns` because it will be fed into the `ggplot` function as `x=columns` to select between the four filtered columns. Finally, the `interact` function is called to visualize the boxplot with the `SelectMultiple` widget.
+
+Read the insights gained from this plot [below](https://ploomber-sql.readthedocs.io/en/latest/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.html#id2)
+
+#### `Seaborn` Scatterplot
+
+After answering some simple questions about the data, we can dive into unravelling complex relationships. The third visualization, therefore, is a seaborn `scatterplot()` to visualize the positive correlation between the recharge time and distance range of electric vehicles. Moreover, by including `hue`s, we can further explore the relationship between the recharge time and distance range of electric vehicles with respect to their size and the year of release. Note that both categorical variables, car size and model year, are discretized to avoid overplotting. If we recall the first bar plot, we saw an increasing rate of growth in unique electric car models between 2021 and 2023. Given this finding, it was feasible to discretize all electric cars' model year data into 2012-2021 and 2022-2023 to obtain a clear understanding of recent technological advancements in car battery and performance. Electric car size has nine unique categories, and are discretized into sedans (or smaller) and SUV's (or larger) to avoid overplotting.
```

Comment:
Same comment as above,please summarize and reduce text, incorporate the visualization

Reply:
amended in latest commit